docs/docs/recipes/working-with-starters.md

```diff
@@ -3,36 +3,36 @@ title: "Recipes: Working with Starters"
 tableOfContentsDepth: 1
 ---
 
-[Starters](/docs/starters/) are boilerplate Gatsby sites maintained officially, or by the community.
+[Starters](/docs/starters/) बॉयलरप्लेट गॅटस्बी साइट आहेत त्यांची अधिकृत देखभाल केली जाते, किंवा समुदायाद्वारे.
 
-## Using a starter
+## स्टार्टर वापरणे
 
-### Prerequisites
+### पूर्व शर्ती
 
-- The [Gatsby CLI](/docs/gatsby-cli) installed
+- द [Gatsby CLI](/docs/gatsby-cli) स्थापित करा
 
-### Directions
+### दिशानिर्देश
 
-1. Find the starter you'd like to use. (_The [Starter Library](/starters/?v=2) is a good place to look!_)
+1. आपण वापरू इच्छित असलेले स्टार्टर शोधा. (द [Starter Library](/starters/?v=2) पाहण्याची चांगली जागा आहे!_)
```

Comment:
remove  द and आहे

Reply:
@spielers I have made all the suggested changes, please merge my PR. 
 Also, Let me know if you need any further changes!

Thanks & Regards,
Abdulhussain Jarif